### PR TITLE
fix: Added adword debug logs

### PR DESCRIPTION
**Pull request by vbmudalige**
### What changes are proposed in this PR?
---
* Added few debug logs for adword debugging purposes

### How was this tested?
---
* Unit tests

### CODE REVIEW

File: hip_data_tools/aws/s3.py [hunk 1]
```diff
@@ -102,6 +102,7 @@ def upload_dataframe_as_parquet(self,
             file_name (str): the name of the file at destination
         Returns: None
         """
+        log.debug("Uploading the dataframe as parquet: \n %s", dataframe.to_string)
```
Comment (arihantsurana) [inline hip_data_tools/aws/s3.py:105]: log the column names, shape and show top 2 rows

File: hip_data_tools/google/adwords.py [hunk 2]
```diff
@@ -209,6 +210,7 @@ def _get_query(self) -> ServiceQueryBuilder:
     def _get_query_pager(self, query: ServiceQueryBuilder):
         if self.__pager is None:
             self.__pager = query.Pager(self._get_service())
+        log.debug("The pager %s", self.__pager)
```
Comment (arihantsurana) [inline hip_data_tools/google/adwords.py:213]: yeah this is not useful

File: hip_data_tools/aws/s3.py [hunk 3]
```diff
@@ -102,6 +102,11 @@ def upload_dataframe_as_parquet(self,
             file_name (str): the name of the file at destination
         Returns: None
         """
+        log.debug("Uploading the dataframe as parquet")
+        log.debug("Column names of the dataframe: %s", list(dataframe))
```
Comment (arihantsurana) [inline hip_data_tools/aws/s3.py:106]: add to the same log statement.

Comment (arihantsurana) [reply hip_data_tools/aws/s3.py:106]: if you have multiple statements, they are prone to get diluted by other logs

Comment (vbmudalige) [reply hip_data_tools/aws/s3.py]: fixed

**Pull request merged**